cypress/support/commands.js

```diff
@@ -43,6 +43,15 @@ Cypress.Commands.add('clickCloseButton', () => {
 Cypress.Commands.add('clickSettingsButton', () => {
   cy.get('label.left').contains('Settings').click()
 })
+
 Cypress.Commands.add('navigateToHomePage', () => {
   cy.visit('http://127.0.0.1:8080')
 })
+
+Cypress.Commands.add('getLeftSoftkeyButton', () => {
+  return cy.get('.left')
```

Comment:
I recommend `cy.get( '.softkey > .left' )` here to get the specific softkey left element, as `.left` may possibly exist in other component

Reply:
fixed and added another small improvement, does it look ok?